Add examples section in the readme (#35)
aaugustin/django-sequences: README.rst

All parameters have the same meaning as in the ``get_next_value`` and
``get_last_value`` functions.

Examples
========

Per-date sequences
------------------

If you want independent sequences per day, month, or year, use the appropriate
date fragment in the sequence name. For example:

.. code:: python

    from django.utils import timezone
    from sequences import get_next_value
    
    # Per-day sequence
    get_next_value(f"books-{timezone.now().date().isoformat()}")
    # Per-year sequence
    get_next_value(f"prototocol-{timezone.now().year}")

The above calls will result in separate sequences like ``books-2023-03-15``

> or ``protocol-2022``, respectively.

Reviewer: ```suggestion
or ``protocol-2022``, respectively.

```

All parameters have the same meaning as in the ``get_next_value`` and
``get_last_value`` functions.

> 

Reviewer: single space between sections
```suggestion
```

Examples
========

> Reset values on a date
> ----------------------

Reviewer: I suggest we don't call this "resetting", since no sequence is being reset. Instead they're independent sequences.

```suggestion
Per-date sequences
------------------
```

> There are situations where you want to reset the next value of a sequence when the date (or month/year) changes. This can be achieved by integrating the date on the name of the sequence. For example:

Reviewer: edits and word wrapping

```suggestion
If you want independent sequences per day, month, or year, use the appropriate
date fragment in the sequence name. For example:
```

.. code:: python

    from django.utils import timezone
    from sequences import get_next_value
    
    # Reset the count every day
    get_next_value(f"books-{timezone.now().date().isoformat()}")
    # or, to reset every year
    get_next_value(f"prototocol-{timezone.now().year}")

> The above will result to creating different sequences named similar to ``books-2023-03-15``, ``books-2023-03-16`` or ``protocol-2022``, ``protocol-2023`` each with their own count.

Reviewer: edits and wrapping

```suggestion
The above calls will result in separate sequences like ``books-2023-03-15``
or ``protocol-2022``, respectively.
```

> 
> 

Reviewer: ```suggestion

```
